units/systemd-resolved.service.in

```diff
@@ -13,6 +13,7 @@ Documentation=man:systemd-resolved.service(8)
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/resolved
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/writing-network-configuration-managers
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/writing-resolver-clients
+DefaultDependencies=no
 After=systemd-networkd.service network.target
 Before=network-online.target nss-lookup.target
```

Comment:
We should also change `Before=network-online.target` by `Before=network.target`. This is primarily important at shutdown, so that resolved is only stopped after the network is not required anymore.